src/main/java/net/pms/logging/LoggingConfig.java

```diff
@@ -0,0 +1,508 @@
+/*
+ * PS3 Media Server, for streaming any medias to your PS3.
+ * Copyright (C) 2010  A.Brochard
```

Comment:
@Nadahar I think we can use the UMS copyright header for this and other new files, since this is mostly new